README.md

```diff
@@ -31,7 +31,9 @@ curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/miniku
 ```
 
 ### Windows
-Download the [minikube-windows-amd64.exe](https://storage.googleapis.com/minikube/releases/latest/minikube-windows-amd64.exe) file, rename it to `minikube.exe` and add it to your path.
+```shell
+choco install minikube
+```
```

Comment:
Could you keep both sets of instructions?

Reply:
@dlorenc Done